kolibri_instant_schools_plugin/assets/src/views/profile-page/index.vue

```diff
@@ -30,18 +30,10 @@
     </section>
 
     <form @submit.prevent="submitEdits">
-      <ui-alert
```

Comment:
Hmm, @indirectlylit had mentioned that we should try to avoid unnecessary changes.